js/app copy.js

```diff
@@ -0,0 +1,177 @@
+let product5 = new Product(`Retro Lonely Breakfast`, `img/breakfast.jpg`);
+let product6 = new Product(`Meatball Bubblegum`, `img/bubblegum.jpg`);
+let product7 = new Product(`Musical Cha-ouch`, `img/chair.jpg`);
+let product8 = new Product(`My First Cthulu, with Sacrifice`, `img/cthulhu.jpg`);
+let product9 = new Product(`Doggie Duckerator`, `img/dog-duck.jpg`);
+let product10 = new Product(`Utili-Pen`, `img/pen.jpg`);
+let product11 = new Product(`Pet Sweep`, `img/pet-sweep.jpg`);
+let product12 = new Product(`Pizzizzors`, `img/scissors.jpg`);
+let product13 = new Product(`Jaws Sleeping Bag`, `img/shark.jpg`);
+let product14 = new Product(`Toddler Swiffer`, `img/sweep.png`);
+let product15 = new Product(`Tauntaun Sleeper`, `img/tauntaun.jpg`);
+let product16 = new Product(`Unicorn Mean`, `img/unicorn.jpg`);
+let product17 = new Product(`Water You Down Can`, `img/water-can.jpg`);
+let product18 = new Product(`Dr Evil Wine Glass`, `img/wine-glass.jpg`);
+
+
+let products = [product1,product2,product3,product4,product5,product6,product7,product8,product9,product10,product11,product12,product13,product14,product15,product16,product17,product18];
+
+// get random goat index
+function getRandomIndex() {
+  // possible return values: 0, 1, 2
+  return Math.floor(Math.random() * products.length)
+}
+
+// render function: invoke function on page load, I want to load 2 random goats
+function renderProducts() {
+  productNumber = [];
+  
+  
+    while(productNumber.length < 3){
+    let randomIndex = getRandomIndex();
+    console.log(`------------randomIndex${randomIndex}`)
+    // while unique numbers does not include the the random generated number inside nums, then push it
+    if(!productNumber.includes(products[randomIndex])){
+    productNumber.push(products[randomIndex]);    
+    }
+  // Otherwise loop until length is 3
+  }
+  console.log(productNumber);
+  
+  let index1 = productNumber.shift();
+  console.log(index1);
+  let index2 = productNumber.shift();
+  console.log(index2);
+  let index3 = productNumber.shift();
+  console.log(index3);
+
+  
+
+  // index1 = getRandomIndex();
+  // index2 = getRandomIndex();
+  // index3 = getRandomIndex();
+  // This prevents the same images from appearing
+
+  // Build while loop for all 3 things********************
+
+  
+  // while (index1 === index2) {
+  //   index2 = getRandomIndex();
+  // }
+
+  // let firstProduct = products[index1];
+  // let secondProduct = products[index2];
+  // let thirdProduct = products[index3];
+
+  // DOM manipulation
+  // This replaces the src for the image with the img src from the goat variable
+  image1.src = index1.src;
+  image1.alt = index1.name;
+  image1.title = index1.name
+  image1.id = index1;
+
+  image2.src = index2.src;
+  image2.alt = index2.name;
+  image2.title = index2.name;
+  image2.id = index2;
+
+  image3.src = index3.src;
+  image3.alt = index3.name;
+  image3.title = index3.name;
+  image3.id = index3;
```

Comment:
Consider doing this in a function you call three times, instead of typing the whole thing out three times.